### DISCUSSION TITLE
Any idea when will v4 be released?

### DISCUSSION THREAD

**Discussion by rodrigocfd** [pmndrs/zustand#983] [Q&A] [answered]
I'm currently using v3 in a project, and I just wrote a quick TypeScript experiment with v4.0.0-rc1 and the built-in `immer` middleware, and it's just **crazy good**.

So, any ideas when the v4 will be official?

**Comment by dai-shi** [comment, marked as answer]
v4 is basically ready, but we would like to get more feedbacks before finalizing.
Especially, because it has `use-sync-external-store` dependency, there seems to be cases with bundling issues.
#953 is a trial, but it doesn't seem to change anything.
https://github.com/facebook/react/pull/24440 is also related.
